ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -318,6 +338,69 @@ Item {
             visible: false
         }
     }
+
+    Repeater {
+      model: imageUrls.split(" ")
+      visible: messageWrapper.appSettings.displayChatImages && imageUrls != ""
+      Rectangle {
+        property int chatVerticalPadding: 7
+        property int chatHorizontalPadding: 12
+
+        id: chatBox2
+        height: {
+          if (!messageWrapper.appSettings.displayChatImages || imageUrls == "") {
+            return 0
+          }
+          return (isCurrentUser || (!isCurrentUser && authorCurrentMsg == authorPrevMsg) ? childrenRect.height : 24 + childrenRect.height)
+        }
+        color: isCurrentUser ? Theme.blue : Theme.lightBlue
+        border.color: Theme.transparent
+        width:  350 + 2 * chatHorizontalPadding
+        radius: 16
+        anchors.left: !isCurrentUser ? chatImage.right : undefined
+        anchors.leftMargin: !isCurrentUser ? 8 : 0
+        anchors.right: !isCurrentUser ? undefined : parent.right
+        anchors.rightMargin: !isCurrentUser ? 0 : Theme.padding
+        anchors.top: (index == 0) ? chatBox.bottom : parent.children[index-1].bottom
+        anchors.topMargin: messageWrapper.appSettings.displayChatImages ? chatBox2.chatVerticalPadding: 0
+        visible: (isMessage || isEmoji) && messageWrapper.appSettings.displayChatImages && imageUrls != ""
+
+        // Thi`s rectangle's only job is to mask the corner to make it less rounded... yep
+        Rectangle {
+            color: parent.color
+            width: 18
+            height: messageWrapper.appSettings.displayChatImages ? 18 : 0
+            anchors.bottom: parent.bottom
+            anchors.bottomMargin: 0
+            anchors.left: !isCurrentUser ? parent.left : undefined
+            anchors.leftMargin: 0
+            anchors.right: !isCurrentUser ? undefined : parent.right
+            anchors.rightMargin: 0
+            radius: 4
+            z: -1
+          }
+
+          Image {
+            id: imageMessage
+            horizontalAlignment: !isCurrentUser ? Text.AlignLeft : Text.AlignRight
+            anchors.left: parent.left
+            anchors.leftMargin: parent.chatHorizontalPadding
+            anchors.top: parent.top
+            anchors.topMargin: messageWrapper.appSettings.displayChatImages ? chatBox2.chatVerticalPadding : 0
+            sourceSize.width: 350
+            source: modelData
+            visible: messageWrapper.appSettings.displayChatImages && imageUrls != ""
+            onStatusChanged: {
+                if (imageMessage.status == Image.Error) {
+                  imageMessage.height = 0
+                  imageMessage.visible = false
+                  chatBox2.height = 0
+                  chatBox2.visible = false
+                }
+            }
```

Comment:
My attempt to "undo" the chatbox for the image if loading the image fails

Reply:
Does the statusChanged function get called? What is not working?

Reply:
Oh yes, this gets called and it's working, just commented here, just in case someone has a better idea to do this